### ISSUE TITLE
Freeplane icons are not found by script or formulas

### ISSUE THREAD

**Issue by Bruno01170** [freeplane/freeplane#1672]
**Describe the bug**
I tried with freeplane the formulas =c.find{it.icons.contains("OK")}.size() to count the number of node that have the OK icons.
But since today, it is not working anymore. It is weird because it is still working with my personal created icons, but not the original one.

**To Reproduce**
1. Create a floating node
2. add on some node the OK icon
3. enter the command =c.find{it.icons.contains("OK")}.size()
4. the number return is always 0

**Expected behavior**
It should be counting the number of OK icons in the mindmap

**Mind map files**
It is on all my mindmap, I guess the problem is more with my freeplane software configuration

**Desktop (please complete the following information):**
 - OS: W11
 - Version 1.11.10
 - Java version: 15.0.10

**Comment by euu2021** [comment]
This is working here:
![image](https://github.com/freeplane/freeplane/assets/77707706/ac91a589-4c41-4bad-b2a3-a52520506360)


Are you aware that this icon is actually called "button_ok"?
![image](https://github.com/freeplane/freeplane/assets/77707706/8ba8d54c-4a46-400c-9283-31678edc7ae7)

**Comment by Bruno01170** [comment]
Yes, it seems that I didn't get the right icon name, but I don't understand why it has worked earlier ...

Anyway, a big thank you, and do you know where I can found the full list of those icons?
Because by hoovering, the name of this one is OK

**Comment by euu2021** [comment]
Returns all the icons that exist in the selected node:
`node.icons.icons`

Another solution: 
the [Developer Tools Add-on](https://docs.freeplane.org/getting-started/Add-ons_(install).html#-developer-tools) has an inspector of menu items that shows information about the icons. For example, when inspecting one icon, it returns this kind of information:
```
Menu item: OK
Key: IconAction.button_ok
Icon Key: IconAction.button_ok.icon
Command line option (copied to clipboard): -XIconAction.button_ok
Path: //main_menu/insert//icons/menu_iconByCategory//icons/IconAction.button_ok
Shortcut: 
```

Another solution:
If you go to `Help->Generate Scripting API map`, there is a node with the icon names:
![image](https://github.com/freeplane/freeplane/assets/77707706/419b2a13-953d-4aed-b597-41267477f8e6)

**Comment by EdoFro** [comment]
another possible solution:

```groovy
=c.find{it.icons.icons.any{it.contains("ok")}}.size()
```

counts all the nodes that have at least one icon which name's contains "ok"
